deploy/osd-route-monitor-operator-reinstall/04-openshift-route-monitor-operator.CronJob.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: sre-operator-reinstall
+  namespace: openshift-route-monitor-operator
+spec:
+  failedJobsHistoryLimit: 1
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Replace
+  schedule: "*/30 * * * *"
+  jobTemplate:
+    spec:
+      ttlSecondsAfterFinished: 100
+      template:
+        spec:
+          serviceAccountName: sre-operator-reinstall-sa
+          restartPolicy: Never
+          containers:
+          - name: operator-reinstaller
+            image: image-registry.openshift-image-registry.svc:5000/openshift/tools:latest
+            imagePullPolicy: Always
+            command:
+            - sh
+            - -c
+            - |
+              #!/bin/bash
+              set -euxo pipefail
+              NAMESPACE=openshift-route-monitor-operator
+              OPERATOR=route-monitor-operator
+
+              # Look for a CSV, then perform the reinstall if found. If you want to _always_ reinstall, remove this check and just keep the logic inside the "if"
+              CSV_FOUND=$(oc -n "$NAMESPACE" get clusterserviceversions.operators.coreos.com -o name | grep "$OPERATOR")
+              if [[ ! -z "$CSV_FOUND" ]]; then
+                # The OLM dance
+                # delete all installplans (if found) (the true is so we don't fail set -e)
+                installplans=$(oc get installplans.operators.coreos.com -n "$NAMESPACE" | grep "$OPERATOR" || true | awk '{print $1}')
+                if [[ -n "$installplans" ]]; then
+                  echo "$installplans" | xargs oc delete installplan.operators.coreos.com -n "$NAMESPACE"
+                fi
```

Comment:
Probably worth updating the SOP with this bit to make it more foolproof

Reply:
Yup, I have a few things I'm going to add